packages/python/javdb_platform/dual_connection.py

```
@@ -298,6 +298,42 @@ def executescript(self, script: str):
+                d1_cursors = batch(statements)
+            else:
+                d1_cursors = [
+                    self._d1.execute(sql, params)
+                    for sql, params in statements
+                ]
+            self._d1_uncommitted_writes += sum(
+                1 for sql, _params in statements if not _is_read(sql)
+            )
+            for sqlite_cur, d1_cur, (sql, _params) in zip(
+                sqlite_cursors, d1_cursors, statements,
+            ):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)